cpp/benchmarks/io/cuio_common.cpp

```diff
@@ -201,3 +201,51 @@ void try_drop_l3_cache()
                            [](auto& cmd) { return exec_cmd(cmd).empty(); }),
                "Failed to execute the drop cache command");
 }
+
+cudf::io::io_type retrieve_io_type_enum(std::string_view io_string)
+{
+  if (io_string == "FILEPATH") {
+    return cudf::io::io_type::FILEPATH;
+  } else if (io_string == "HOST_BUFFER") {
+    return cudf::io::io_type::HOST_BUFFER;
+  } else if (io_string == "DEVICE_BUFFER") {
+    return cudf::io::io_type::DEVICE_BUFFER;
+  } else if (io_string == "VOID") {
+    return cudf::io::io_type::VOID;
+  } else if (io_string == "USER_IMPLEMENTED") {
+    return cudf::io::io_type::USER_IMPLEMENTED;
+  } else {
+    CUDF_FAIL("Unsupported io_type.");
```

Comment:
Since you always call `return`, don't need any `else`.
```suggestion
  if (io_string == "FILEPATH") { return cudf::io::io_type::FILEPATH; } 
  if (io_string == "HOST_BUFFER") { return cudf::io::io_type::HOST_BUFFER; }
  if (io_string == "DEVICE_BUFFER") ...
  ....
  CUDF_FAIL("Unsupported io_type.");
```

Reply:
Got it!